### ISSUE TITLE
Adapt new version of kalkun to last version of gammu 1.39

### ISSUE THREAD

**Issue by SingeLiren** [back2arie/Kalkun#183]
I'm using the Gammu version 1.39, Kalkun version 7.1 work fine but there is problem with database because     'pbk' table were removed from new version of Gammu so no possibility to work with phone-book...  Any ideas to correct this issue?

**Comment by tenzap** [comment]
Gammu SMSD DB schema removed unused daemons, pbk and pbk_groups tables in schema v16.
The commands bellow restore theses tables so that they can be used by kalkun

More details here:
* https://wammu.eu/docs/manual/smsd/tables.html#history-of-database-structure
* https://github.com/gammu/gammu/commit/07cda98944793144dcd0fb1dd12321be1ea0a0b3#diff-e0d7ca5ebbe8dcbe1f17ef77adc90a29
* https://github.com/gammu/gammu/commit/6203631d6d4f4286d1b69033ba83a65b48484fff#diff-e0d7ca5ebbe8dcbe1f17ef77adc90a29

Please, note these SQL statements are for PostgreSQL only. If you use mySQL just have a look a the commits above to find their mysql version.

```
--
-- Sequence declarations for tables' primary keys
--

--CREATE SEQUENCE pbk_groups_ID_seq;

-- --------------------------------------------------------

--
-- Index declarations for tables' primary keys
--

--CREATE UNIQUE INDEX pbk_groups_pkey ON pbk_groups USING btree ("ID");

-- --------------------------------------------------------
-- 
-- Table structure for table "daemons"
-- 

CREATE TABLE daemons (
  "Start" text NOT NULL,
  "Info" text NOT NULL
);

-- 
-- Dumping data for table "daemons"
-- 


-- --------------------------------------------------------

-- 
-- Table structure for table "pbk"
-- 

CREATE TABLE pbk (
  "ID" serial PRIMARY KEY,
  "GroupID" integer NOT NULL DEFAULT '-1',
  "Name" text NOT NULL,
  "Number" text NOT NULL
);

-- 
-- Dumping data for table "pbk"
-- 


-- --------------------------------------------------------

-- 
-- Table structure for table "pbk_groups"
-- 

CREATE TABLE pbk_groups (
  "Name" text NOT NULL,
  "ID" serial PRIMARY KEY
);

-- 
-- Dumping data for table "pbk_groups"
-- 


-- --------------------------------------------------------
```

**Comment by tenzap** [comment]
For MySQL it seems the change is already in upstream

**Comment by tenzap** [comment]
fixed by pr #194  for postgresql

**Comment by SingeLiren** [comment]
Thank you very much!
